layouts/partials/head.html

```diff
@@ -39,10 +39,6 @@
 {{ template "_internal/google_analytics_async.html" . }}
 {{ end }}
 {{ partialCached "head-css.html" . "asdf" }}
-<script
-  src="{{ "js/jquery-3.3.1.min.js" | relURL }}"
-  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
```

Comment:
Are we losing this `integrity` value from the replacement code?

Reply:
added integrity SRIs